### PR TITLE
docs: move contribution out of version

### PR DESCRIPTION
**Pull request by zirain**
xref: https://github.com/envoyproxy/gateway/issues/2987

### CODE REVIEW

File: site/content/zh/_index.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ title: Envoy Gateway
 {{% /blocks/feature %}}
 
 {{% blocks/feature icon="fab fa-github" title="欢迎贡献！"
-    url="/latest/contributions/" %}}
+    url="/contributions/" %}}
 We do a [Pull Request](https://github.com/envoyproxy/gateway/pulls) contributions workflow on **GitHub**.
 我们在 **GitHub** 通过 [Pull Request](https://github.com/envoyproxy/gateway/pulls) 开启贡献流程。
```
Comment (wilsonwu) [inline site/content/zh/_index.md:71]:
```suggestion
我们在 **GitHub** 通过 [Pull Request](https://github.com/envoyproxy/gateway/pulls) 开启贡献流程。
```

**Review by wilsonwu — CHANGES_REQUESTED** [review]
Seems it was my mistake for this translation, please help to remove this line.

**Review by Xunzhuo — APPROVED** [review]
+1 for this, thanks for working on it : )

**Review by arkodg — APPROVED** [review]
LGTM thanks !
feels good to delete lines, doesnt it ?

**Comment by zirain** [comment]
> LGTM thanks ! feels good to delete lines, doesnt it ?

next is the `releases` folder, haha!

**Pull request merged**